src/main/java/nl/beehivejava/scriptj/PluginEnvironment.java

```diff
@@ -0,0 +1,12 @@
+package nl.beehivejava.scriptj;
+
+import nl.beehivejava.scriptj.script.ScriptType;
+
+/**
+ * @author Lesley
+ */
+public interface PluginEnvironment {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "PluginEnvironment" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1609)